frontend/src/global_styles/content/work_packages/inplace_editing/_display_fields.sass

```diff
@@ -126,6 +126,8 @@ display-field
   &.combinedDate, &.date
     span
       vertical-align: middle
+      @supports (-moz-appearance: none)
```

Comment:
Why is the `moz-appearance` relevant? If you want to check for the browser, maybe our helper classes would be better, e.g `-browser-firefox`